README.md

```diff
@@ -121,6 +121,21 @@ const credential = await navigator.credentials.create({
 
 See the [Guide to Web Authentication](https://webauthn.guide/) for mode details about the `navigator.credentials` API.
 
+#### Creating a PaymentCredential in a cross-origin iframe
+
+Unlike normal WebAuthn credentials, a PaymentCredential is allowed to be created in a
+cross-origin iframe (e.g. if `merchant.com` embeds an iframe from `bank.com`). This is
+intended to support the common enrollment flow of a bank enrolling the user during a
+step-up challenge (e.g. after proving their identity via OTP).
+
+To allow this, the cross-origin iframe must have `allow="payment"` set:
```

Comment:
Nit: Technically speaking, `allow="payment"` is only one way to set permission policy, so stating "must have `allow="payment"`" can be misleading. Why not state that the "iframe must have the "payment" permission policy set, for example:"... and then follow up with the `<iframe src="https://example.org" allow="payment">` example as you have below. Do you think that would be more accurate?

Reply:
Not a nit at all; a learning lesson for me (I didn't know about the other ways to set permission policy). Done.